libshaderc_spvc/include/spvc/spvc.hpp

```diff
@@ -361,6 +362,45 @@ class Context {
     return shaderc_spvc_set_decoration(context_.get(), id, decoration, argument);
   }
 
+  // Get spirv_cross decoration (added for GLSL API support in Dawn).
+  // Given an id and a decoration, result is sent out through |argument|
+  // if |id| does not exist, returns an error message.
+  shaderc_spvc_status GetDecoration(uint32_t id,
+                                    shaderc_spvc_decoration decoration,
+                                    uint32_t* argument){
```

Comment:
nit: ) { instead of ){. Possibly need to run clang format, since this is occuring below?

Reply:
For some reason the format short-cut that I have sat-up in my IDE always ignores this file!

Reply:
Possibly the fact it is a .hpp?